quinn/examples/common.rs

```diff
@@ -0,0 +1,79 @@
+//! Commonly used code in most examples.
+
+use failure::Error;
```

Comment:
Could uses of this here and in other examples be replaced with `Box<std::error::Error>`? failure seems to be on the way out, so I'd prefer not to introduce new uses of it.

Reply:
> failure seems to be on the way out
You mean you're switching to smth else for error handling?

Reply:
We stopped using failure internally some time ago.